dynawo/sources/Models/CPP/Controls/Voltage/ModelVoltageMeasurementsUtilities/DYNModelVoltageMeasurementsUtilities.cpp

```diff
@@ -0,0 +1,348 @@
+//
+// Copyright (c) 2015-2022, RTE (http://www.rte-france.com)
+// See AUTHORS.txt
+// All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at http://mozilla.org/MPL/2.0/.
+// SPDX-License-Identifier: MPL-2.0
+//
+// This file is part of Dynawo, an hybrid C++/Modelica open source suite of simulation tools
+// for power systems.
+//
+
+/**
+ * @file  DYNModelVoltageMeasurementsUtilities.cpp
+ *
+ * @brief Measurements utility model
+ *
+ * All measurements of the network are connected to this model
```

Comment:
Seems unclear: I would propose "Model to compute some measurements on the voltage of connected models. Only running models will be taken into account during computation"